models/agent_model.go

```diff
@@ -263,7 +264,9 @@ func (s *Agent) DSN(service *Service, dialTimeout time.Duration, database string
 		return u.String()
 
 	case PostgresExporterType, QANPostgreSQLPgStatementsAgentType:
-		q := make(url.Values)
+		var pgCon []string
```

Comment:
@AlekSi could you remind me why did we decide to use URI format instead of key=value format?

Reply:
To use URLs everywhere and to avoid problems with spaces on pmm-agent side – command-line flags with spaces may be problematic.

But why we do that change there?

Reply:
Re-read linked issues. Let's use `postgres://username:password@unix/database?host=/var/run/postgresql` for UNIX sockets only

Reply:
Yes, I have tried such implementation based on https://github.com/lib/pq/issues/796  - it looks messier (especially`` with encoding) and has more conditions.
On the other hand, the key-value form is more readable and easy to compose.
Both URI and Key-values forms are quite widely used.

> To use URLs everywhere and to avoid problems with spaces on pmm-agent side – command-line flags with spaces may be problematic.

I considered these points. The first one is reasonable enoght to change back to URI form.

> Re-read linked issues. Let's use postgres://username:password@unix/database?host=/var/run/postgresql for UNIX sockets only

@AlekSi, could you please point me where you found `postgres://username:password@**unix**/database?host=/var/run/postgresql` such format.

I have seen only:
` postgres+pg8000://<db_user>:<db_pass>@/<db_name>?unix_sock=/cloudsql/<cloud_sql_instance_name>/.s.PGSQL.5432`

Reply:
> could you please point me where you found postgres://username:password@unix/database?host=/var/run/postgresql such format.

In the same issue you linked

Reply:
> Re-read linked issues. Let's use postgres://username:password@unix/database?host=/var/run/postgresql for UNIX sockets only

Unfortunately, "unix" as a stub for the host does not work.

```
INFO[2020-05-21T11:00:50.648+00:00] CheckConnectionRequest: type:POSTGRESQL_SERVICE dsn:"postgres://postgres@unix/postgres?connect_timeout=2&host=%2Fvar%2Frun%2Fpostgresql%2F&sslmode=disable" timeout:<seconds:3 > .  request=552641f0-9b52-11ea-8f2c-0242ac1f0202
INFO[2020-05-21T11:00:50.649+00:00] CheckConnection response: error:"connector: &{map[client_encoding:UTF8 connect_timeout:2 datestyle:ISO, MDY dbname:postgres extra_float_digits:2 host:unix port:5432 sslmode:disable user:postgres] {{0 {0 0 <nil>} <nil> false 0 0 <nil> <nil> <nil>}}}; error: <nil>" .  request=552641f0-9b52-11ea-8f2c-0242ac1f0202
```